public/css/common.less

```diff
@@ -410,5 +410,6 @@ form[name="form_confirm_removal"] {
   background-color: @state-critical;
   color: @text-color-inverted;
   padding: 0 0.25em;
+  margin-left: .28125em + .28125em; // 0 calculated &nbsp; width;
```

Comment:
why two times?

please make this dependent on where it's located, this is layout after all